src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -167,13 +167,14 @@ p5.RendererGL.prototype.vertex = function(x, y) {
 
 /**
  * Sets the normal to use for subsequent vertices.
- * @method vertexNormal
+ * @private
```

Comment:
I was initially wondering why this was `@private`, but for anyone else reading, it's because we already have public docs for `normal()` here: https://github.com/processing/p5.js/blob/v1.9.0/src/core/shape/vertex.js#L1107

This is just an internal method on the renderer that was accidentally producing a public doc before.